lib/brain.js

```diff
@@ -3142,10 +3142,13 @@ Brain.prototype._dispense = function _dispense () {
       if (!this.billDispenser.initialized) {
         const cassettes = this.trader.cassettes
 
-        return this.billDispenser.init({
-          cassettes,
-          fiatCode: this.trader.locale.fiatCode
-        })
+        // wait for port to close
+        setTimeout(() => {
+          return this.billDispenser.init({
+            cassettes,
+            fiatCode: this.trader.locale.fiatCode
+          })
+        }, 200)
```

Comment:
Can you explain in more detail why we need this delay and why specifically 200ms?